### ISSUE TITLE
Label Image Decoding and Saving

### ISSUE THREAD

**Issue by Eggotron** [rawilk/laravel-ups#1]
Hey rawilk! Amazing package you have here, really helpful and the documentation is very understandable. This "issue" is more of a "feature wishlist" if you would like that kind of input :)

On the shipping class there are labels of the packages generated on the process, this "package" returns an encoded image but the documentation doesn't explain how to convert and save that to the database and project. I solved it like this:

```
 /* Cleaning the code */
 $image = base64_decode($packages->first()->label_image->graphic_image);

 /* Saving code as an image on a folder in the system */
 file_put_contents('img/tracking-labels/order_' . $order->id . '.gif', $image);

 /* Saving the name of the file */
 $order_package->package_label_file = 'order_' . $order->id . '.gif';
```

Pretty straightforward. Would be great that this package aids on the decoding of the image and just returning the file to save it on a default folder on a single line, like calling a class or something. Just an idea.

Awesome work, thank you.

**Comment by rawilk** [comment]
@Eggotron Thanks for the idea. That sounds like a pretty common use case, and I'll probably implement something like that into the package at some point. I'm not sure when I'll get a chance to work on it, but I'll definitely consider adding it.

**Comment by rawilk** [comment]
This kind of functionality is now available in version `2.1.0`.

To get the decoded image contents, just call: `$package->getDecodedImageContent()` on a package result.

To have this package store the image for you, you can call `$package->storeLabel()`. This will store the image on the configured file system disk, and will return the name of the file.

**Comment by rawilk** [comment]
Also, sorry for not getting to this sooner; I just haven't had a chance to maintain this package until now.
